pillar/era-articles.yaml

```diff
@@ -19,3 +19,8 @@
     display: https://elife.stencila.io/article-52258/
     # snapshot number missing just for launch
     download: https://hub.stenci.la/api/projects/583/snapshots/13/archive
+
+'43154':
+    date: 2020-10-22T10:30:00Z'
```

Comment:
Missing quotation

Reply:
Whoops! Thanks for catching updated in latest commit